chrome/content/indicator.jsm

```diff
@@ -93,7 +92,19 @@ var SPDYManager = {
         if (!spdyHeader || !spdyHeader.length) return;
 
         // find the browser which this request originated from
-        let domWindow = getLoadContext(subject);
+        let loadContext = getLoadContext(subject);
+        if (!loadContext) return null;
```

Comment:
Apologies for the nitpick, but this `return null` is inconsistent with the other return statements in this function.